abi-wrapper/Cargo.toml

```diff
@@ -15,4 +15,4 @@ cpi = ["no-entrypoint"]
 [dependencies]
 anchor-lang = "0.26.0"
 anchor-spl = "0.26.0"
-zeta-abi = "0.1.1"
+zeta-abi = { path = "../" }
```

Comment:
- [ ] todo change back to 1.0.0 once approved and crate is published

Reply:
we don't need to change this back? can leave as is